### PR TITLE
Add test for all endpoints

### PR DESCRIPTION
**Pull request by efuaakum**
Add environment variables to manage local and test version of todos.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,23 @@
...
-  }
-```
+Ensure your finished repository meets all of the [Assessment Criteria](#assessment-criteria) listed below
 
 ## Assessment Criteria
 
-1. The root currently serves the file `index.html`. The `index.html` file should outline the expected endpoints for the service.
+1. The root URL should serve/return the file `index.html`. The `index.html` file should outline the expected endpoints for the service.
 
 1. The `index.html` file should reference a `favicon.ico`, `logo.png` and a `styles.css` file.
```
Comment (tanyapowell) [inline README.md:15]: What is the `logo.png` for?

File: README.md [hunk 2]
```diff
@@ -33,13 +27,69 @@ The API should `CREATE`, `READ`, `UPDATE` and `DELETE` from a list of todos in a
...
+```javascript
+fs.writeFile(__dirname + process.env.BASE_JSON_PATH,  content, err => {
+    if (err) {
+      console.error(err)
+      return
+    }
+}
+```
+
+2. Clone this repository to your local machine. Once cloned, open in Visual Code. Run the following commands to install any dependencies:
```
Comment (tanyapowell) [inline README.md:74]: Does this step need to come before step 1?

File: src/index.js [hunk 3]
```diff
@@ -1,54 +1,51 @@
...
 app.use(express.json());
 app.use(express.raw());
+app.use(bodyParser.json());
 
 app.use("/content", express.static(path.join(__dirname, "public")));
 
 app.get("/", (_, res) => {
+  /*
   res.sendFile("./public/index.html", { root: __dirname });
+  */
```
Comment (tanyapowell) [inline src/index.js:24]: Should this be mentioned in the README, so they don't delete it?

File: README.md (hunk 1 above)
Comment (efuaakum) [reply README.md:15]: Added logo

File: README.md (hunk 2 above)
Comment (efuaakum) [reply README.md:74]: Yes, good shout. I'll move it now

**Review by tanyapowell — APPROVED** [review]

**Pull request merged**